kolibri/plugins/setup_wizard/assets/src/vue/index.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+
+  <div class="device-owner-creation">
+    <div class="wrapper">
+      <img class="logo" src="./icons/logo-min.png">
+      <div class="container">
+        <div class="title">Device Owner</div>
+        <br>
+        <div class="description">To use Kolibri, there are need to be at least one device owner. This account will have the highest privilege that can configure all settings about this installation.</div>
```

Comment:
"To use Kolibri, you first need to create a Device Owner account. This account will be used to configure high-level settings for this installation, and create other administrator accounts."